README.md

```diff
@@ -36,6 +36,27 @@ Run API token Rails generator and follow instructions:
 ```bash
 bundle exec rails generate webflow_sync:api_token_flow
 ```
+### Configuration options
+
+In `config/initializers/webflow_sync.rb` you can specify configuration options:
+
+1. `webflow_site_id`
+2. `skip_webflow_sync` - skip synchronization for different environments
+3. `sync_webflow_slug` - save slug generated on WebFlow on the model record
```

Comment:
> save slug generated on WebFlow to the Rails database, in the Rails model column. This can be useful if you want to link to WebFlow item directly from your Rails app:

```rb
link_to('View on site', "https://#{webflow_domain}/articles/#{record.webflow_slug}", target: :blank)
```